src/Pages/Shared/ConfigMapSecret/ConfigMapSecretProtected.tsx

```diff
@@ -184,46 +183,47 @@ export const ConfigMapSecretProtected = ({
     )
 
     const renderApproveButton = () => {
```

Comment:
Query: Can i edit the draft after approving, if yes, will my approval be valid?

Reply:
You have already discussed ig